src/main/scala/no/ndla/searchapi/service/search/SearchConverterService.scala

```diff
@@ -987,9 +1008,13 @@ trait SearchConverterService {
     private def getTopicTaxonomyContexts(topic: Topic,
                                          filterVisibles: Boolean,
                                          bundle: TaxonomyBundle): Try[List[SearchableTaxonomyContext]] = {
-      val topicsConnections = bundle.topicResourceConnections.filter(_.resourceId == topic.id)
-      val topics = bundle.topics.filter(topic => topicsConnections.map(_.topicid).contains(topic.id)) :+ topic
-      val parentTopicsAndPaths = topics.flatMap(t => getParentTopicsAndPaths(t, bundle, List(t.id)))
```

Comment:
Her blei det forsøkt å hente ut en liste med connections fra ressurser, nokke som alltid var tom liste. For å bøte på det blei topic slengt på på slutten. Ny kode sjekker faktisk korrekt koblingsobjektliste for å finne parents.